apiConfig.js

```diff
@@ -0,0 +1,125 @@
+const BASE_API_URL = 'https://fakerestapi.azurewebsites.net/api/v1';
+
+const urls = {
+    activities : {
+        getAll : {
+            method : 'GET',
+            url : `${BASE_API_URL}/Activities`
```

Comment:
We should either remove `method` properties, since they aren't used, or rename the whole object from `urls` to something like `endpoints`

Reply:
done